### ISSUE TITLE
Possibility of running hugo in-browser so users don't have to download it

### ISSUE THREAD

**Issue by Xyncgas** [gohugoio/hugo#8972]
We can have many more exciting grounds to stand upon, like online-collaborations, design on the fly so it's accessible in the classroom without installations, export the result in a zip, or a single html file that compact all the stuffs you made including javascript and html...

**Comment by jmooring** [comment]
I'm not even sure where to begin with this, except to say... no.

**Comment by bep** [comment]
>I'm not even sure where to begin with this, except to say... no.

@jmooring running Hugo (or at least the core parts of it) in the browser (e.g. via WebAssembly) should be possible, and maybe something we will eventually do (as part of a CMS integration) ... But that was most likely not the intention of this thread starter.

**Comment by Xyncgas** [comment]
Yeah maybe u are right, because I've forgot what I meant when I was asking this question heheh~
Although I'd love to be having hugo available as a nuget package, then it'd be great since I can render websites dynamically in the browser with blazor with the pages generated by hugo.
But sorry for not being able to give you guys a clearer answers and thanks for looking at this proposal
